packages/libs/react-ui/src/components/Form/NumberField/NumberField.tsx

```diff
@@ -151,14 +151,14 @@ export function NumberFieldBase(
 
         <div className={buttonContainerClass}>
           <Button
-            icon={<ChevronUp size="sm" className={iconClass} />}
+            icon={<MonoExpandLess size="sm" className={iconClass} />}
             variant="text"
             isCompact
             className={buttonClass}
             {...incrementButtonProps}
           />
           <Button
-            icon={<ChevronDown size="sm" className={iconClass} />}
+            icon={<MonoExpandMore className={iconClass} />}
```

Comment:
check style is broken

Reply:
We force the size in the new button component.